SonarQube.MSBuild.Tasks/WriteProjectInfoFile.cs

```diff
@@ -268,15 +263,12 @@ private Property TryCreateSettingFromItem(ITaskItem taskItem)
 
             Property setting = null;
 
-            string settingId;
-
-            if (TryGetSettingId(taskItem, out settingId))
+            if (TryGetSettingId(taskItem, out string settingId))
             {
                 // No validation for the value: can be anything, but the
                 // "Value" metadata item must exist
-                string settingValue;
 
-                if (TryGetSettingValue(taskItem, out settingValue))
+                if (TryGetSettingValue(taskItem, out string settingValue))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1066)